db/datastore.py

```diff
@@ -275,3 +275,39 @@ def get_followers(self):
             return None
         else:
             return proto[0]
+
+class MessageStore(object):
+    def __init__(self):
+        self.db = lite.connect(DATABASE)
+        self.db.text_factory = str
+        try:
+            cursor = self.db.cursor()
+            cursor.execute('''CREATE TABLE messages(guid BLOB , signed_pubkey BLOB,
+encryption_pubkey BLOB, subject TEXT, message_type TEXT, message TEXT, timestamp, INTEGER, signature BLOB)''')
+            cursor.execute('''CREATE INDEX idx1 ON messages(guid);''')
+            self.db.commit()
+        except Exception:
+            pass
```

Comment:
I think this goes back to the talk about db schema creation and also migrations going forward. Maybe we should extract this out to some db specific stuff.